services/galley/fake-sqs.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+function cleanup() {
+  printf 'cleaning up ...'
+  docker kill $(cat DOCKER_ID)
+  rm DOCKER_ID
+  printf done
+}
+
+trap cleanup EXIT
+
+docker run -d --cidfile DOCKER_ID -p 4568:4568 airdock/fake-sqs 
+sleep 2
+curl http://localhost:4568 -d "Action=CreateQueue&QueueName=integration-team-events.fifo" 
```

Comment:
Instead of a `sleep n`, could we instead run `curl` until it succeeds, sth like
```
while true
do
  curl --silent http://localhost:4568 -d "Action=CreateQueue&QueueName=integration-team-events.fifo"
  if [ $? -eq 0 ]; then
    echo "Queue created, all set!"
    break
  else
    echo "Waiting for the queue to be created..."
  fi
  sleep 1
done
```

Reply:
Good idea, thanks!